workloads/1-topic-1-partition-100b.yaml

```diff
@@ -22,6 +22,7 @@ name: 1 topic / 1 partition / 1Kb
 topics : 1
 partitionsPerTopic : 1
 messageSize : 100
+payloadFile: "payload/payload-100b.data"
```

Comment:
Should we rather have only one between messageSize and payloadFile? Otherwise it's not immediately clear which one will take effect.

Reply:
I was thinking about this but I think this can be done separate from this change set